app/assets/stylesheets/app/consent_form_preview.scss

```diff
@@ -0,0 +1,38 @@
+.consent-agreement {
```

Comment:
Is a generic name or way we could do this?  Could you just use p.highlight?

I only say these things because I come from the school of thought with CSS styles that you create everything as a re-usable component from the start, especially as I want to extract the styles into their own npm package and GEM so I can retrofit them to FGM